src/BenchmarksApps/HttpClientBenchmarks/Clients/HttpClient/ClientOptionsBinder.cs

```diff
@@ -15,6 +15,7 @@ public class ClientOptionsBinder : BinderBase<ClientOptions>
         public static Option<int> ConcurrencyPerHttpClientOption { get; } = new Option<int>("--concurrencyPerHttpClient", () => 1, "Number of concurrect requests per one HttpClient");
         public static Option<int> Http11MaxConnectionsPerServerOption { get; } = new Option<int>("--http11MaxConnectionsPerServer", () => 0, "Max number of HTTP/1.1 connections per server, 0 for unlimited");
         public static Option<bool> Http20EnableMultipleConnectionsOption { get; } = new Option<bool>("--http20EnableMultipleConnections", () => true, "Enable multiple HTTP/2.0 connections");
+        public static Option<bool> Http30EnableMultipleConnectionsOption { get; } = new Option<bool>("--http30EnableMultipleConnections", () => true, "Enable multiple HTTP/3.0 connections");
```

Comment:
I wonder if we maybe should keep the default as false for the time being.
Otherwise if we merge it like that, all the CI tests will start using that by default, but I think it should be a clear parameter as with H/2 (h2 vs h2multiconn)
It's not a blocker, just a way not to mix up measurements.

Or we can change both crank config yml and ci test config yml right away to include the parameter

Reply:
???
Do you want me to change something here or not?

Reply:
**(1)**
"Keep the default as false" -- Minimal change
```diff
- ... new Option<bool>(..., () => true, ...);
+ ... new Option<bool>(..., () => false, ...);
```

-OR-

**(2)**
"Change crank config yml" -- Modify [scenarios/httpclient.benchmarks.yml](https://github.com/aspnet/Benchmarks/blob/9c454b7357ac0b032c48535df236b8bbb9588052/scenarios/httpclient.benchmarks.yml) to include new parameter as a variable (with default value = false), e.g.: https://github.com/aspnet/Benchmarks/blob/9c454b7357ac0b032c48535df236b8bbb9588052/scenarios/httpclient.benchmarks.yml#L27
and pass it to client, e.g.: `--http20EnableMultipleConnections {{http20EnableMultipleConnections}}`
https://github.com/aspnet/Benchmarks/blob/9c454b7357ac0b032c48535df236b8bbb9588052/scenarios/httpclient.benchmarks.yml#L171

-OR-

**(3)**
"Change both crank config yml and ci test config yml" -- (2) AND modify [build/httpclient-scenarios.yml](https://github.com/aspnet/Benchmarks/blob/9c454b7357ac0b032c48535df236b8bbb9588052/build/httpclient-scenarios.yml) to include new parameter with both true and false to test matrix, e.g.:
https://github.com/aspnet/Benchmarks/blob/9c454b7357ac0b032c48535df236b8bbb9588052/build/httpclient-scenarios.yml#L31-L34
and as `httpVersion` is already passed to the scenario (we're _not_ adding a new dimension), it should then appear automatically in "pretty graphs" as soon as the new value will appear in the result table.

---

> Do you want me to change something here or not?

It is hard for me to understand how urgently you'd like to merge the changes (I guess not really, given the PR's in draft), and how much effort are you willing to put into it.

My _preferred_ option is (3) but it _can_ be done as a follow-up, as long as the default is false.

But I am open to the discussion about the default as well.

Reply:
> It is hard for me to understand how urgently you'd like to merge the changes

Not urgent at all, it just wasn't obvious to me from the original comment whether you expect any changes to this PR or not.